src/thd_initdblk.c

```diff
@@ -1124,18 +1141,15 @@ ENTRY("THD_datablock_apply_atr") ;
       /* load oblique transformation matrix */
       if(atr_flt) {
           if( validate ){               /* disco change */
-              
+
               /* just testing here */
-              float fracdiff , old , new;
-              
-              old = 1;
-              new = 2;
-              
-              if( fabs(old) + fabs(new) != 0 ){
-                  fracdiff = fabs( old - new ) / ( fabs(old) + fabs(new) );
-                  printf("fracdiff %13.6f\n",fracdiff);
-              }
-                            
+              float fracdiff;
+
+              fracdiff = frac_diff(atr_flt->fl[0],
+                                   dset->daxes->ijk_to_dicom_real.m[0][0]);
```

Comment:
I don't think the float var here should have the same name as the function---it makes my head hurt.  (Perhaps it should make the compiler's head hurt, too.)